ansible/roles/test/tasks/fib.yml

```diff
@@ -63,4 +63,4 @@
       - fib_info='/root/fib_info.txt'
       - ipv4={{ipv4}}
       - ipv6={{ipv6}}
-    ptf_extra_options: "--relax --debug info --log-file /tmp/fib_test.FibTest.ipv4.{{ipv4}}.ipv6.{{ipv6}}.{{ansible_date_time.iso8601}}.log "
+    ptf_extra_options: "--platform-dir ptftests --relax --debug info --log-file /tmp/fib_test.FibTest.ipv4.{{ipv4}}.ipv6.{{ipv6}}.{{ansible_date_time.iso8601}}.log "
```

Comment:
this is not needed since we've already having the `ptf_platform_dir: ptftests`

Reply:
Thanks, I removed the extra option.

Also, I don't have to exclude port 42 anymore. The cabling issue has been addressed by the lab team.

Thanks,
Ying